### PR TITLE
Very minimal empty ODF files

### PR DESCRIPTION
**Pull request by timar**
These files contain no data at all. This guarantees that when the
user opens a new document in the richdocuments app, the language
of paragraph, page size, cell date format, currency, etc. will be
according to the current locale.

### CODE REVIEW

**Comment by mmattel** [comment]
@phil-davis can you retrigger codecov, seems to be stuck...

**Comment by phil-davis** [comment]
codecov/patch is always stuck these days - it always says "In progress - This step has started..." and no result comes. I have had to override the result for every PR to core in the last week or more.

**Comment by phil-davis** [comment]
@micbar please get the appropriate person to review this...

**Comment by micbar** [comment]
@phil-davis @DeepDiver1975 Background? Changelog?

**Comment by mmattel** [comment]
@timar can you please file a changelog, see: https://github.com/owncloud/core/blob/master/changelog/README.md

**Comment by phil-davis** [comment]
> @phil-davis @DeepDiver1975 Background? Changelog?

I don't know - I just got mentioned regarding codecov.

File: changelog/unreleased/37896 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Bugfix: Add very minimal empty ODF files
+
+These files contain no data at all. This guarantees that when the
+user opens a new document in the richdocuments app, the language
+of paragraph, page size, cell date format, currency, etc. will be
+according to the current locale.
+
+Previous files were in German. E.g. document language (spellcheck
+language) was set to German, spreasheet tab name was "Tabelle",
```
Comment (phil-davis) [inline changelog/unreleased/37896:9]:
```suggestion
language) was set to German, spreadsheet tab name was "Tabelle",
```

File: changelog/unreleased/37896 [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+etc. It was not right for non-German users.
```
Comment (mmattel) [inline changelog/unreleased/37896:10]: `It was not right for non-German users` --> `It was incorrect for non-German users`

**Review by mmattel — APPROVED** [review]
LGTM (but not tested with the richdocument app because not installed)

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Stupid codecov. I tried to fake the results, but it comes in "over the top" with its own silly result.

@micbar please force merge.

**Comment by mmattel** [comment]
@micbar ping (force merge, see comment of @phil-davis above)

**Pull request merged**